src/Discord/CommandClient/Command.php

```diff
@@ -62,19 +76,23 @@ class Command
      * @param \Callable            $callable    The callable function.
      * @param string               $description The description of the command.
      * @param string               $usage       The usage of the command.
+     * @param int                  $cooldown    The usage of the command.
```

Comment:
I guess this should be described as "The cooldown of the command in seconds" or sth?